content/docs/2.6/scalers/azure-app-insights.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Azure Application Insights"
+layout = "scaler"
+availability = "v1.3+"
+maintainer = "Community"
+description = "Scale applications based on Azure Application Insights metrics."
+go_file = "azure_app_insights_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-app-insights` trigger that scales based on an Azure Application Insights metric.
+
+```yaml
+triggers:
+- type: azure-app-insights
+  metadata:
+    metricAggregationTimespan: "0:1"
+    metricAggregationType: avg
+    metricFilter: cloud/roleName eq 'role_name'
+    metricName: "metric_name"
+    metricNamespace: "customMetrics"
+    targetValue: "1"
+    activeDirectoryClientIdFromEnv: CLIENT_ID_ENV_NAME # Optional, can use TriggerAuthentication as well
+    activeDirectoryClientPasswordFromEnv: CLIENT_PASSWORD_ENV_NAME # Optional, can use TriggerAuthentication as well
+    appIdFromEnv: APP_ID # Optional, can use TriggerAuthentication as well
+    tenantIdFromEnv: TENANT_ID` # Optional, can use TriggerAuthentication as well
+```
+
+This scaler is backed by the Azure Application Instance REST API. Please see [this](https://dev.applicationinsights.io/reference) page
+for further details.
+
+**Parameter list:**
```

Comment:
Can you align with our guidelines please? https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler

Reply:
Hopefully, this is better. Please let me know if the doc is still not aligned.

Reply:
Thank you!